src/Raven.Server/Documents/Indexes/Persistence/Lucene/IndexWriteOperation.cs

```diff
@@ -131,7 +131,7 @@ public virtual void IndexDocument(LazyStringValue key, object document, Indexing
                 stats.RecordIndexingOutput();
 
                 if (_logger.IsInfoEnabled)
-                    _logger.Info($"Indexed document for '{_indexName}'. Key: {key}. Output: {_converter.Document}.");
+                    _logger.Info($"Indexed document for '{_indexName}'. Key: {key}.");
```

Comment:
This still generates an entry per doc, no?

Reply:
I'll investigate the logs after this change, but the issue with reduce was with the actual output, which was VERY long.

Reply:
That's how we behave in 3.5 when logging level is Debug - we put entire index entries to logs.  Do we no longer want that?

Reply:
No, see an example output in the issue. One log line with very long...

Reply:
I think not. We might want to add an endpoint that give us an index entry, or something like that, but not beyond it.